packages/terra-text/tests/jest/Text.test.jsx

```diff
@@ -109,4 +109,10 @@ describe('Text', () => {
       expect(e.message).toContain('The prop `children` is marked as required');
     }
   });
+
+  it('should forward ref to span', () => {
+    const ref = React.createRef();
+    mount(<React.Fragment><Text id="TestId" ref={ref}>Test</Text></React.Fragment>);
```

Comment:
This may look weird. I referred to https://github.com/airbnb/enzyme/issues/1852#issuecomment-433145879 to wrap the original component while testing.